### DISCUSSION TITLE
Sliver DNS Listener issues

### DISCUSSION THREAD

**Discussion by dunit42** [BishopFox/sliver#1542] [Q&A]
Hello,

I have been trying to get Sliver DNS C2 channel working as a test. I had to run the following commands on my Ubuntu system, which fixed one DNS issue I was having (listed in the docs):

systemctl disable systemd-resolved.service
systemctl stop systemd-resolved
rm -f /etc/resolv.conf
vim /etc/resolv.conf

This allowed me to start the DNS listener on my machine (WINNING!). I purchased a domain with NameCheap, and created the following DNS records:

Type             Host                                 Value
A record        domain-name.xyz           My IP address
A record        ns1.domain-name.xyz     My IP address
NS record     1.domain-name.xyz         ns1.my-domain.xyz

However, when I start the DNS listener, and run the EXE on a test windows machine, I do not get an active session. I have forwarded TCP & UDP ports 53 to my sliver C2 system (that is required, right?). I have previously used TLS beacons with no issue. Does anyone know what I might be doing wrong?

Thanks for any guidance.

**Comment by omgt2k** [comment]
Hi @dunit42,
I'm encountering a similar problem and saw your previous post about it. Did you manage to find a solution? I've been searching but haven't had any luck yet. Any insights you could share would be greatly appreciated!

Thanks
